wiki/Ranking_Criteria/en.md

```diff
@@ -3,7 +3,11 @@ Ranking Criteria
 
 For the record, these include the recent changes made [from the discussion forum](https://osu.ppy.sh/forum/87). Any rule that is being discussed is listed here as it used to be and will be updated once the discussion reaches an agreement.
 
-**Keep in mind that the [Code of Conduct](/wiki/Modding_Guides/Modding_and_Mapping) is part of the Ranking Criteria and applies to all existent game modes in _osu!_**
+**Keep in mind that the [Code of Conduct](/wiki/Code_of_Conduct) is part of the Ranking Criteria and applies to all existent game modes in _osu!_**
+
+_For mode specific criteria, see [osu!](/wiki/rc_osu!), [osu!taiko](/wiki/rc_osu!taiko), [osu!catch](/wiki/rc_osu!catch), and [osu!mania](/wiki/rc_osu!mania)._
+
+_For skinning related criteria, see [Skin Set List](/wiki/Skin_Set_List)._
```

Comment:
the skinning related criteria is not in effect yet, also shouldn't it be /wiki/Ranking_Criteria/Skin_Set_List ?
I would just remove that entire link to that for now.